Dockerfile

```diff
@@ -79,5 +79,10 @@ ENV GROQ_API_KEY=${GROQ_API_KEY} \
     VITE_LOG_LEVEL=${VITE_LOG_LEVEL} \
     DEFAULT_NUM_CTX=${DEFAULT_NUM_CTX}
 
-RUN mkdir -p ${WORKDIR}/run
-CMD pnpm run dev --host
+# Create run directory with proper permissions
+RUN mkdir -p /app/run && chown node:node /app/run
+
```

Comment:
directory was created but files were copied to the /app directory during copy stage. do you need to put this line at the start with a followed by a chdir command?